aikau/src/main/resources/alfresco/services/DialogService.js

```diff
@@ -479,7 +481,8 @@ define(["dojo/_base/declare",
             contentWidth: payload.contentWidth ? payload.contentWidth : null,
             contentHeight: payload.contentHeight ? payload.contentHeight : null,
             handleOverflow: handleOverflow,
-            fixedWidth: fixedWidth
+            fixedWidth: fixedWidth,
+            noMinWidth: payload.noMinWidth || false
```

Comment:
One thing you could consider here that would enforce a boolean for truthy as well as falsy values would be to have `!!payload.noMinWidth`. Also applies to line 729 below.